### PR TITLE
Added TS0603

### PR DESCRIPTION
**Pull request by reifl**

### CODE REVIEW

File: src/devices/tuya.ts [hunk 1]
```diff
@@ -5503,6 +5503,31 @@ const definitions: Definition[] = [
...
+                [12, null, null],
+            ],
+        },
+        description: 'Garage door opener',
+        configure: async (device, coordinatorEndpoint) => {
+            await tuya.configureMagicPacket(device, coordinatorEndpoint);
+            const endpoint = device.getEndpoint(1);
+            await reporting.bind(endpoint, coordinatorEndpoint, ['genBasic']);
+        },
+        fromZigbee: [legacy.fromZigbee.matsee_garage_door_opener, fz.ignore_basic_report, tuya.fz.datapoints],
```
Comment (Koenkk) [inline src/devices/tuya.ts:5525]:
```suggestion
        fromZigbee: [tuya.fz.datapoints],
```

Can you use this and add everything to `meta. tuyaDatapoints` instead?

(same for `toZigbee`)

File: src/devices/tuya.ts [hunk 2]
```diff
@@ -5503,6 +5503,31 @@ const definitions: Definition[] = [
             e.binary('garage_door_contact', ea.STATE, false, true)
                 .withDescription('Indicates if the garage door contact is closed (= true) or open (= false)')],
     },
+    {
+        fingerprint: [{modelID: 'TS0603', manufacturerName: '_TZE608_c75zqghm'}],
+        zigbeeModel: ['TS0603'],
+        model: 'TS0603',
+        vendor: '_TZE608_c75zqghm',
+        extend: [],
```
Comment (Koenkk) [inline src/devices/tuya.ts:5511]:
```suggestion
```

Can be removed

File: src/devices/tuya.ts [hunk 3]
```diff
@@ -5503,6 +5503,31 @@ const definitions: Definition[] = [
             e.binary('garage_door_contact', ea.STATE, false, true)
                 .withDescription('Indicates if the garage door contact is closed (= true) or open (= false)')],
     },
+    {
+        fingerprint: [{modelID: 'TS0603', manufacturerName: '_TZE608_c75zqghm'}],
+        zigbeeModel: ['TS0603'],
+        model: 'TS0603',
+        vendor: '_TZE608_c75zqghm',
```
Comment (Koenkk) [inline src/devices/tuya.ts:5510]:
```suggestion
        vendor: 'TuYa',
```

I guess it's TuYa?

File: src/devices/tuya.ts [hunk 4]
```diff
@@ -5503,6 +5503,31 @@ const definitions: Definition[] = [
             e.binary('garage_door_contact', ea.STATE, false, true)
                 .withDescription('Indicates if the garage door contact is closed (= true) or open (= false)')],
     },
+    {
+        fingerprint: [{modelID: 'TS0603', manufacturerName: '_TZE608_c75zqghm'}],
+        zigbeeModel: ['TS0603'],
```
Comment (Koenkk) [inline src/devices/tuya.ts:5508]:
```suggestion
```

Not needed because of `fingerprint`

**Comment by reifl** [comment]
sorry, wasn't at home last week so i can't change and test it. Hope i didn't miss something

File: src/devices/tuya.ts [hunk 5]
```diff
@@ -5503,6 +5503,30 @@ const definitions: Definition[] = [
             e.binary('garage_door_contact', ea.STATE, false, true)
                 .withDescription('Indicates if the garage door contact is closed (= true) or open (= false)')],
     },
+    {
+        fingerprint: [{modelID: 'TS0603', manufacturerName: '_TZE608_c75zqghm'}],
+        model: 'TS0603',
```
Comment (Koenkk) [inline src/devices/tuya.ts:5508]: - Could you provide a link to this device?
- Do you have a better model number? Is there something written on the device?

Comment (reifl) [reply src/devices/tuya.ts:5508]: bought it on aliexpress: https://aliexpress.com/item/1005006776475188.html
the only text on the Device is QS-Zigbee-C03 but all i found on this name is a curtain motor controller.

Comment (Koenkk) [reply src/devices/tuya.ts:5508]: Then let's leave it as is, thanks!

**Pull request merged**